data/api/server-server/transactions.yaml

```diff
@@ -46,64 +35,70 @@ paths:
       parameters:
         - in: path
           name: txnId
-          type: string
-          description: |-
-            The transaction ID.
-          required: true
-          x-example: S0meTransacti0nId
-        - in: body
-          name: body
-          type: object
+          description: The transaction ID.
           required: true
+          example: S0meTransacti0nId
           schema:
-            allOf:
-              - $ref: "definitions/transaction.yaml"
-              - type: object
-                properties:
-                  edus:
-                    type: array
-                    description: |-
-                      List of ephemeral messages. May be omitted if there are no ephemeral
-                      messages to be sent. Must not include more than 100 EDUs.
-                    items:
-                      $ref: "definitions/edu.yaml"
-          example: {
-            "$ref": "examples/transaction.json",
-            "edus": [{"$ref": "examples/edu.json"}]
-          }
```

Comment:
Another lost example. Also, `$ref` cannot be combined with `edus` this way but this is already fixed in the most recent `main`.